src/main/java/org/broadinstitute/hellbender/utils/hmm/segmentation/HiddenStateSegment.java

```diff
@@ -75,17 +75,21 @@ public double getEndQuality() {
 
     /**
      * Returns the "event quality" (NDP): 1 - posterior probability that the hidden
-     * state is the copy {@link CopyNumberTriState#NEUTRAL neutral} across the whole segment
+     * state is different from the the "reference state" across the whole segment
```

Comment:
It seems that with your change in phrasing the `1 - . . .` of the previous line no longer applies.